datashader_cli: add page (#10373)
tldr-pages/tldr: pages/common/datashader_cli.md

# datashader_cli

> Quick visualization of large datasets using CLI based on datashader.
> More information: <https://github.com/wybert/datashader-cli>.

> - Create a shaded scatter plot of points and save it to a png file, set the background color to black:
> 
> `datashader_cli points {{path/to/input.parquet}} --x {{pickup_x}} --y {{pickup_y}} {{path/to/output.png}} --background black`

Reviewer: ```suggestion
- Create a shaded scatter plot of points and save it to a png file and set the background color:

`datashader_cli points {{path/to/input.parquet}} --x {{pickup_x}} --y {{pickup_y}} {{path/to/output.png}} --background {{black|white|#rrggbb}}`
```

> - Visualize the geospatial data, support Geoparquet, shapefile, geojson, geopackage, etc:
> 
> `datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo true`

Reviewer: ```suggestion
- Visualize the geospatial data (supports Geoparquet, shapefile, geojson, geopackage, etc.):

`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo {{true}}`
```

> `datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo {{true}}`

Reviewer 2: ```suggestion
`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo true`
```

> # datashader_cli
> 
> > Quick visualization of large datasets using CLI based on [datashader](https://github.com/holoviz/datashader).
> > More information: <https://github.com/wybert/datashader-cli>.
> 
> - Create a shaded scatter plot of points and save it to png file, set background color to black:
> 
> `datashader_cli points nyc_taxi.parquet --x pickup_x --y pickup_y pickup-scatter.png --background black`
> 
> - Visualize the geospaital data, support Geoparquet, shapefile, geojson, geopackage, etc:
> 
> `datashader_cli points data.geo.parquet data.png --geo true`
> 
> - Use matplotlib to render the image:
> 
> `datashader_cli points data.geo.parquet data.png --geo true --matplotlib true`

Reviewer 2: Thanks for your contribution, I have some suggestions for this page.
```suggestion
# datashader_cli

> Quick visualization of large datasets using CLI based on datashader.
> More information: <https://github.com/wybert/datashader-cli>.

- Create a shaded scatter plot of points and save it to a png file, set the background color to black:

`datashader_cli points {{path/to/input.parquet}} --x {{pickup_x}} --y {{pickup_y}} {{path/to/output.png}} --background black`

- Visualize the geospatial data, support Geoparquet, shapefile, geojson, geopackage, etc:

`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo true`

- Use matplotlib to render the image:

`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo {{true}} --matplotlib true`
```

> `datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo {{true}} --matplotlib true`

Reviewer: ```suggestion
`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo {{true}} --matplotlib {{true}}`
```
